opendevin/memory/memory.py

```diff
@@ -1,5 +1,13 @@
 import threading
 
+try:
+    # workaround for GitHub Codespaces
+    __import__('pysqlite3')
+    import sys
+    sys.modules['sqlite3'] = sys.modules.pop('pysqlite3')
+except:
+    pass
```

Comment:
This is not very clean, so if possible it would be nice to avoid this workaround, as I'm a bit worried about the maintenance burden (it may cause side-effects). Could you explain the problem and suggest some other alternatives.

Reply:
Chroma requires SQLite > 3.35 but SQLite in GitHub codespace is less than that.